modules/end-user-guide/pages/viewing-language-servers-and-debug-adapters-logs-in-visual-studio-code.adoc

```diff
@@ -0,0 +1,20 @@
+:_content-type: ASSEMBLY
+:description: Viewing {prod-short} workspace language servers and debug adapters logs in Visual Studio Code
+:keywords: administration-guide, logs
+:navtitle: Language servers and debug adapters logs in the editor
+:page-aliases:
+
+[id="viewing-language-servers-and-debug-adapters-logs-in-visual-studio-code"]
+= Viewing language servers and debug adapters logs in Visual Studio Code
+
+In the Visual Studio Code editor running for your workspace,
+you can view language servers and debug adapters extensions logs.
+
+.Procedure
+. Configure the extension: click File > Preferences > Settings, expand the Extensions section, search for your extension, and set the `trace.server` or similar configuration to `verbose`, if such configuration exists.
```

Comment:
```suggestion
. Configure the extension: click *File* > *Preferences* > *Settings*, expand the *Extensions* section, search for your extension, and set the `trace.server` or similar configuration to `verbose`, if such configuration exists.
```

Reply:
I think this might work better when split into multiple tasks:
. Click *File* > *Preferences* > *Settings*.
. Expand the *Extensions* section to search for your extension.
. Set the `trace.server` or similar configuration to `verbose`, if such configuration exists.